### ISSUE TITLE
nc_test failing in 4.6.0

### ISSUE THREAD

**Issue by ArchangeGabriel** [Unidata/netcdf-c#808]
Hi,

ArchLinux packager here, got a test failure.

### Environment Information

* What platform are you using? (please provide specific distribution/version in summary)
  * [x] Linux
* 32 and/or 64 bit?
  * [x] 64-bit
* What build system are you using?
  * [x] cmake

### Summary of Issue

```
53/159 Testing: nc_test
53/159 Test: nc_test
Command: "/build/netcdf/src/build/nc_test/nc_test"
Directory: /build/netcdf/src/build/nc_test
"nc_test" start time: Jan 25 22:14 CET
Output:
----------------------------------------------------------
Testing 5 different netCDF formats.


Switching to netCDF classic format.
*** testing nc_strerror ... ok
*** testing nc_open ... ok
*** testing nc_close ... ok
*** testing nc_inq ... ok
*** testing nc_inq_dimid ... ok
*** testing nc_inq_dim ... ok
*** testing nc_inq_dimlen ... ok
*** testing nc_inq_dimname ... ok
*** testing nc_inq_varid ... ok
*** testing nc_inq_var ... ok
*** testing nc_inq_natts ... ok
*** testing nc_inq_ndims ... ok
*** testing nc_inq_nvars ... ok
*** testing nc_inq_unlimdim ... ok
*** testing nc_inq_vardimid ... ok
*** testing nc_inq_varname ... ok
*** testing nc_inq_varnatts ... ok
*** testing nc_inq_varndims ... ok
*** testing nc_inq_vartype ... ok
*** testing nc_get_var_text ... ok
*** testing nc_get_var_uchar ... ok
*** testing nc_get_var_schar ... ok
*** testing nc_get_var_short ... ok
*** testing nc_get_var_int ... ok
*** testing nc_get_var_long ... ok
*** testing nc_get_var_float ... ok
*** testing nc_get_var_double ... ok
*** testing nc_get_var_ushort ... ok
*** testing nc_get_var_uint ... ok
*** testing nc_get_var_longlong ... ok
*** testing nc_get_var_ulonglong ... ok
*** testing nc_get_var1_text ... ok
*** testing nc_get_var1_uchar ... ok
*** testing nc_get_var1_schar ... ok
*** testing nc_get_var1_short ... ok
*** testing nc_get_var1_int ... ok
*** testing nc_get_var1_long ... ok
*** testing nc_get_var1_float ... ok
*** testing nc_get_var1_double ... ok
*** testing nc_get_var1_ushort ... ok
*** testing nc_get_var1_uint ... ok
*** testing nc_get_var1_longlong ... ok
*** testing nc_get_var1_ulonglong ... ok
*** testing nc_get_var1 ... ok
*** testing nc_get_vara_text ... ok
*** testing nc_get_vara_uchar ... ok
*** testing nc_get_vara_schar ... ok
*** testing nc_get_vara_short ... ok
*** testing nc_get_vara_int ... ok
*** testing nc_get_vara_long ... ok
*** testing nc_get_vara_float ... ok
*** testing nc_get_vara_double ... ok
*** testing nc_get_vara_ushort ... ok
*** testing nc_get_vara_uint ... ok
*** testing nc_get_vara_longlong ... ok
*** testing nc_get_vara_ulonglong ... ok
*** testing nc_get_vara ... ok
*** testing nc_get_vars_text ... ok
*** testing nc_get_vars_uchar ... ok
*** testing nc_get_vars_schar ... ok
*** testing nc_get_vars_short ... ok
*** testing nc_get_vars_int ... ok
*** testing nc_get_vars_long ... ok
*** testing nc_get_vars_float ... ok
*** testing nc_get_vars_double ... ok
*** testing nc_get_vars_ushort ... ok
*** testing nc_get_vars_uint ... ok
*** testing nc_get_vars_longlong ... ok
*** testing nc_get_vars_ulonglong ... ok
*** testing nc_get_vars ... ok
*** testing nc_get_varm_text ... ok
*** testing nc_get_varm_uchar ... ok
*** testing nc_get_varm_schar ... ok
*** testing nc_get_varm_short ... ok
*** testing nc_get_varm_int ... ok
*** testing nc_get_varm_long ... ok
*** testing nc_get_varm_float ... ok
*** testing nc_get_varm_double ... ok
*** testing nc_get_varm_ushort ... ok
*** testing nc_get_varm_uint ... ok
*** testing nc_get_varm_longlong ... ok
*** testing nc_get_varm_ulonglong ... ok
*** testing nc_get_varm ... ok
*** testing nc_get_att_text ... 
	FAILURE at line 10854 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_get.c: value read not that expected
	### 1 FAILURES TESTING nc_get_att_text! ###
*** testing nc_get_att_uchar ... ok
*** testing nc_get_att_schar ... ok
*** testing nc_get_att_short ... ok
*** testing nc_get_att_int ... ok
*** testing nc_get_att_long ... ok
*** testing nc_get_att_float ... ok
*** testing nc_get_att_double ... ok
*** testing nc_get_att_ushort ... ok
*** testing nc_get_att_uint ... ok
*** testing nc_get_att_longlong ... ok
*** testing nc_get_att_ulonglong ... ok
*** testing nc_get_att ... 
	FAILURE at line 1689 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_read.c: buf read not that expected
	### 1 FAILURES TESTING nc_get_att! ###
*** testing nc_inq_att ... ok
*** testing nc_inq_attname ... ok
*** testing nc_inq_attid ... ok
*** testing nc_inq_attlen ... ok
*** testing nc_inq_atttype ... ok
*** testing nc_create ... ok
*** testing nc_redef ...  1386 good comparisons.  34 good comparisons.  1386 good comparisons.  34 good comparisons. ok
*** testing nc_sync ...  34 good comparisons.  1386 good comparisons. ok
*** testing nc_abort ...  1386 good comparisons.  34 good comparisons. ok
*** testing nc_def_dim ... ok
*** testing nc_rename_dim ... ok
*** testing nc_def_var ... ok
*** testing nc_put_var_text ... ok
*** testing nc_put_var_uchar ... ok
*** testing nc_put_var_schar ... ok
*** testing nc_put_var_short ... ok
*** testing nc_put_var_int ... ok
*** testing nc_put_var_long ... ok
*** testing nc_put_var_float ... ok
*** testing nc_put_var_double ... ok
*** testing nc_put_var_ushort ... ok
*** testing nc_put_var_uint ... ok
*** testing nc_put_var_longlong ... ok
*** testing nc_put_var_ulonglong ... ok
*** testing nc_put_var1_text ... ok
*** testing nc_put_var1_uchar ... ok
*** testing nc_put_var1_schar ... ok
*** testing nc_put_var1_short ... ok
*** testing nc_put_var1_int ... ok
*** testing nc_put_var1_long ... ok
*** testing nc_put_var1_float ... ok
*** testing nc_put_var1_double ... ok
*** testing nc_put_var1_ushort ... ok
*** testing nc_put_var1_uint ... ok
*** testing nc_put_var1_longlong ... ok
*** testing nc_put_var1_ulonglong ... ok
*** testing nc_put_var1 ...  1386 good comparisons. ok
*** testing nc_put_vara_text ... ok
*** testing nc_put_vara_uchar ... ok
*** testing nc_put_vara_schar ... ok
*** testing nc_put_vara_short ... ok
*** testing nc_put_vara_int ... ok
*** testing nc_put_vara_long ... ok
*** testing nc_put_vara_float ... ok
*** testing nc_put_vara_double ... ok
*** testing nc_put_vara_ushort ... ok
*** testing nc_put_vara_uint ... ok
*** testing nc_put_vara_longlong ... ok
*** testing nc_put_vara_ulonglong ... ok
*** testing nc_put_vara ...  1386 good comparisons. ok
*** testing nc_put_vars_text ... ok
*** testing nc_put_vars_uchar ... ok
*** testing nc_put_vars_schar ... ok
*** testing nc_put_vars_short ... ok
*** testing nc_put_vars_int ... ok
*** testing nc_put_vars_long ... ok
*** testing nc_put_vars_float ... ok
*** testing nc_put_vars_double ... ok
*** testing nc_put_vars_ushort ... ok
*** testing nc_put_vars_uint ... ok
*** testing nc_put_vars_longlong ... ok
*** testing nc_put_vars_ulonglong ... ok
*** testing nc_put_vars ...  1386 good comparisons. ok
*** testing nc_put_varm_text ... ok
*** testing nc_put_varm_uchar ... ok
*** testing nc_put_varm_schar ... ok
*** testing nc_put_varm_short ... ok
*** testing nc_put_varm_int ... ok
*** testing nc_put_varm_long ... ok
*** testing nc_put_varm_float ... ok
*** testing nc_put_varm_double ... ok
*** testing nc_put_varm_ushort ... ok
*** testing nc_put_varm_uint ... ok
*** testing nc_put_varm_longlong ... ok
*** testing nc_put_varm_ulonglong ... ok
*** testing nc_put_varm ...  1386 good comparisons. ok
*** testing nc_rename_var ...  1386 good comparisons. ok
*** testing nc_put_att_text ... ok
*** testing nc_put_att_uchar ... ok
*** testing nc_put_att_schar ... ok
*** testing nc_put_att_short ... ok
*** testing nc_put_att_int ... ok
*** testing nc_put_att_long ... ok
*** testing nc_put_att_float ... ok
*** testing nc_put_att_ulonglong ... ok
*** testing nc_put_att_ushort ... ok
*** testing nc_put_att_uint ... ok
*** testing nc_put_att_longlong ... ok
*** testing nc_put_att_ulonglong ... ok
*** testing nc_put_att ... 
	FAILURE at line 1085 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/util.c: nc_get_att_text: unexpected value
 33 good comparisons. 
	### 1 FAILURES TESTING nc_put_att! ###
*** testing nc_copy_att ...  34 good comparisons. ok
*** testing nc_rename_att ... 
	FAILURE at line 1859 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_write.c: get_att_text: unexpected value
	### 1 FAILURES TESTING nc_rename_att! ###
*** testing nc_del_att ... ok
*** testing nc_against_pnetcdf ... ok
*** testing nc_set_default_format ... ok


Switching to 64-bit offset format.
*** testing nc_strerror ... ok
*** testing nc_open ... ok
*** testing nc_close ... ok
*** testing nc_inq ... ok
*** testing nc_inq_dimid ... ok
*** testing nc_inq_dim ... ok
*** testing nc_inq_dimlen ... ok
*** testing nc_inq_dimname ... ok
*** testing nc_inq_varid ... ok
*** testing nc_inq_var ... ok
*** testing nc_inq_natts ... ok
*** testing nc_inq_ndims ... ok
*** testing nc_inq_nvars ... ok
*** testing nc_inq_unlimdim ... ok
*** testing nc_inq_vardimid ... ok
*** testing nc_inq_varname ... ok
*** testing nc_inq_varnatts ... ok
*** testing nc_inq_varndims ... ok
*** testing nc_inq_vartype ... ok
*** testing nc_get_var_text ... ok
*** testing nc_get_var_uchar ... ok
*** testing nc_get_var_schar ... ok
*** testing nc_get_var_short ... ok
*** testing nc_get_var_int ... ok
*** testing nc_get_var_long ... ok
*** testing nc_get_var_float ... ok
*** testing nc_get_var_double ... ok
*** testing nc_get_var_ushort ... ok
*** testing nc_get_var_uint ... ok
*** testing nc_get_var_longlong ... ok
*** testing nc_get_var_ulonglong ... ok
*** testing nc_get_var1_text ... ok
*** testing nc_get_var1_uchar ... ok
*** testing nc_get_var1_schar ... ok
*** testing nc_get_var1_short ... ok
*** testing nc_get_var1_int ... ok
*** testing nc_get_var1_long ... ok
*** testing nc_get_var1_float ... ok
*** testing nc_get_var1_double ... ok
*** testing nc_get_var1_ushort ... ok
*** testing nc_get_var1_uint ... ok
*** testing nc_get_var1_longlong ... ok
*** testing nc_get_var1_ulonglong ... ok
*** testing nc_get_var1 ... ok
*** testing nc_get_vara_text ... ok
*** testing nc_get_vara_uchar ... ok
*** testing nc_get_vara_schar ... ok
*** testing nc_get_vara_short ... ok
*** testing nc_get_vara_int ... ok
*** testing nc_get_vara_long ... ok
*** testing nc_get_vara_float ... ok
*** testing nc_get_vara_double ... ok
*** testing nc_get_vara_ushort ... ok
*** testing nc_get_vara_uint ... ok
*** testing nc_get_vara_longlong ... ok
*** testing nc_get_vara_ulonglong ... ok
*** testing nc_get_vara ... ok
*** testing nc_get_vars_text ... ok
*** testing nc_get_vars_uchar ... ok
*** testing nc_get_vars_schar ... ok
*** testing nc_get_vars_short ... ok
*** testing nc_get_vars_int ... ok
*** testing nc_get_vars_long ... ok
*** testing nc_get_vars_float ... ok
*** testing nc_get_vars_double ... ok
*** testing nc_get_vars_ushort ... ok
*** testing nc_get_vars_uint ... ok
*** testing nc_get_vars_longlong ... ok
*** testing nc_get_vars_ulonglong ... ok
*** testing nc_get_vars ... ok
*** testing nc_get_varm_text ... ok
*** testing nc_get_varm_uchar ... ok
*** testing nc_get_varm_schar ... ok
*** testing nc_get_varm_short ... ok
*** testing nc_get_varm_int ... ok
*** testing nc_get_varm_long ... ok
*** testing nc_get_varm_float ... ok
*** testing nc_get_varm_double ... ok
*** testing nc_get_varm_ushort ... ok
*** testing nc_get_varm_uint ... ok
*** testing nc_get_varm_longlong ... ok
*** testing nc_get_varm_ulonglong ... ok
*** testing nc_get_varm ... ok
*** testing nc_get_att_text ... 
	FAILURE at line 10854 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_get.c: value read not that expected
	### 1 FAILURES TESTING nc_get_att_text! ###
*** testing nc_get_att_uchar ... ok
*** testing nc_get_att_schar ... ok
*** testing nc_get_att_short ... ok
*** testing nc_get_att_int ... ok
*** testing nc_get_att_long ... ok
*** testing nc_get_att_float ... ok
*** testing nc_get_att_double ... ok
*** testing nc_get_att_ushort ... ok
*** testing nc_get_att_uint ... ok
*** testing nc_get_att_longlong ... ok
*** testing nc_get_att_ulonglong ... ok
*** testing nc_get_att ... 
	FAILURE at line 1689 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_read.c: buf read not that expected
	### 1 FAILURES TESTING nc_get_att! ###
*** testing nc_inq_att ... ok
*** testing nc_inq_attname ... ok
*** testing nc_inq_attid ... ok
*** testing nc_inq_attlen ... ok
*** testing nc_inq_atttype ... ok
*** testing nc_create ... ok
*** testing nc_redef ...  1386 good comparisons.  34 good comparisons.  1386 good comparisons.  34 good comparisons. ok
*** testing nc_sync ...  34 good comparisons.  1386 good comparisons. ok
*** testing nc_abort ...  1386 good comparisons.  34 good comparisons. ok
*** testing nc_def_dim ... ok
*** testing nc_rename_dim ... ok
*** testing nc_def_var ... ok
*** testing nc_put_var_text ... ok
*** testing nc_put_var_uchar ... ok
*** testing nc_put_var_schar ... ok
*** testing nc_put_var_short ... ok
*** testing nc_put_var_int ... ok
*** testing nc_put_var_long ... ok
*** testing nc_put_var_float ... ok
*** testing nc_put_var_double ... ok
*** testing nc_put_var_ushort ... ok
*** testing nc_put_var_uint ... ok
*** testing nc_put_var_longlong ... ok
*** testing nc_put_var_ulonglong ... ok
*** testing nc_put_var1_text ... ok
*** testing nc_put_var1_uchar ... ok
*** testing nc_put_var1_schar ... ok
*** testing nc_put_var1_short ... ok
*** testing nc_put_var1_int ... ok
*** testing nc_put_var1_long ... ok
*** testing nc_put_var1_float ... ok
*** testing nc_put_var1_double ... ok
*** testing nc_put_var1_ushort ... ok
*** testing nc_put_var1_uint ... ok
*** testing nc_put_var1_longlong ... ok
*** testing nc_put_var1_ulonglong ... ok
*** testing nc_put_var1 ...  1386 good comparisons. ok
*** testing nc_put_vara_text ... ok
*** testing nc_put_vara_uchar ... ok
*** testing nc_put_vara_schar ... ok
*** testing nc_put_vara_short ... ok
*** testing nc_put_vara_int ... ok
*** testing nc_put_vara_long ... ok
*** testing nc_put_vara_float ... ok
*** testing nc_put_vara_double ... ok
*** testing nc_put_vara_ushort ... ok
*** testing nc_put_vara_uint ... ok
*** testing nc_put_vara_longlong ... ok
*** testing nc_put_vara_ulonglong ... ok
*** testing nc_put_vara ...  1386 good comparisons. ok
*** testing nc_put_vars_text ... ok
*** testing nc_put_vars_uchar ... ok
*** testing nc_put_vars_schar ... ok
*** testing nc_put_vars_short ... ok
*** testing nc_put_vars_int ... ok
*** testing nc_put_vars_long ... ok
*** testing nc_put_vars_float ... ok
*** testing nc_put_vars_double ... ok
*** testing nc_put_vars_ushort ... ok
*** testing nc_put_vars_uint ... ok
*** testing nc_put_vars_longlong ... ok
*** testing nc_put_vars_ulonglong ... ok
*** testing nc_put_vars ...  1386 good comparisons. ok
*** testing nc_put_varm_text ... ok
*** testing nc_put_varm_uchar ... ok
*** testing nc_put_varm_schar ... ok
*** testing nc_put_varm_short ... ok
*** testing nc_put_varm_int ... ok
*** testing nc_put_varm_long ... ok
*** testing nc_put_varm_float ... ok
*** testing nc_put_varm_double ... ok
*** testing nc_put_varm_ushort ... ok
*** testing nc_put_varm_uint ... ok
*** testing nc_put_varm_longlong ... ok
*** testing nc_put_varm_ulonglong ... ok
*** testing nc_put_varm ...  1386 good comparisons. ok
*** testing nc_rename_var ...  1386 good comparisons. ok
*** testing nc_put_att_text ... ok
*** testing nc_put_att_uchar ... ok
*** testing nc_put_att_schar ... ok
*** testing nc_put_att_short ... ok
*** testing nc_put_att_int ... ok
*** testing nc_put_att_long ... ok
*** testing nc_put_att_float ... ok
*** testing nc_put_att_ulonglong ... ok
*** testing nc_put_att_ushort ... ok
*** testing nc_put_att_uint ... ok
*** testing nc_put_att_longlong ... ok
*** testing nc_put_att_ulonglong ... ok
*** testing nc_put_att ... 
	FAILURE at line 1085 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/util.c: nc_get_att_text: unexpected value
 33 good comparisons. 
	### 1 FAILURES TESTING nc_put_att! ###
*** testing nc_copy_att ...  34 good comparisons. ok
*** testing nc_rename_att ... 
	FAILURE at line 1859 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_write.c: get_att_text: unexpected value
	### 1 FAILURES TESTING nc_rename_att! ###
*** testing nc_del_att ... ok
*** testing nc_against_pnetcdf ... ok
*** testing nc_set_default_format ... ok


Switching to netCDF-4 format (with NC_CLASSIC_MODEL).
*** testing nc_strerror ... ok
*** testing nc_open ... ok
*** testing nc_close ... ok
*** testing nc_inq ... ok
*** testing nc_inq_dimid ... ok
*** testing nc_inq_dim ... ok
*** testing nc_inq_dimlen ... ok
*** testing nc_inq_dimname ... ok
*** testing nc_inq_varid ... ok
*** testing nc_inq_var ... ok
*** testing nc_inq_natts ... ok
*** testing nc_inq_ndims ... ok
*** testing nc_inq_nvars ... ok
*** testing nc_inq_unlimdim ... ok
*** testing nc_inq_vardimid ... ok
*** testing nc_inq_varname ... ok
*** testing nc_inq_varnatts ... ok
*** testing nc_inq_varndims ... ok
*** testing nc_inq_vartype ... ok
*** testing nc_get_var_text ... ok
*** testing nc_get_var_uchar ... ok
*** testing nc_get_var_schar ... ok
*** testing nc_get_var_short ... ok
*** testing nc_get_var_int ... ok
*** testing nc_get_var_long ... ok
*** testing nc_get_var_float ... ok
*** testing nc_get_var_double ... ok
*** testing nc_get_var_ushort ... ok
*** testing nc_get_var_uint ... ok
*** testing nc_get_var_longlong ... ok
*** testing nc_get_var_ulonglong ... ok
*** testing nc_get_var1_text ... ok
*** testing nc_get_var1_uchar ... ok
*** testing nc_get_var1_schar ... ok
*** testing nc_get_var1_short ... ok
*** testing nc_get_var1_int ... ok
*** testing nc_get_var1_long ... ok
*** testing nc_get_var1_float ... ok
*** testing nc_get_var1_double ... ok
*** testing nc_get_var1_ushort ... ok
*** testing nc_get_var1_uint ... ok
*** testing nc_get_var1_longlong ... ok
*** testing nc_get_var1_ulonglong ... ok
*** testing nc_get_var1 ... ok
*** testing nc_get_vara_text ... ok
*** testing nc_get_vara_uchar ... ok
*** testing nc_get_vara_schar ... ok
*** testing nc_get_vara_short ... ok
*** testing nc_get_vara_int ... ok
*** testing nc_get_vara_long ... ok
*** testing nc_get_vara_float ... ok
*** testing nc_get_vara_double ... ok
*** testing nc_get_vara_ushort ... ok
*** testing nc_get_vara_uint ... ok
*** testing nc_get_vara_longlong ... ok
*** testing nc_get_vara_ulonglong ... ok
*** testing nc_get_vara ... ok
*** testing nc_get_vars_text ... ok
*** testing nc_get_vars_uchar ... ok
*** testing nc_get_vars_schar ... ok
*** testing nc_get_vars_short ... ok
*** testing nc_get_vars_int ... ok
*** testing nc_get_vars_long ... ok
*** testing nc_get_vars_float ... ok
*** testing nc_get_vars_double ... ok
*** testing nc_get_vars_ushort ... ok
*** testing nc_get_vars_uint ... ok
*** testing nc_get_vars_longlong ... ok
*** testing nc_get_vars_ulonglong ... ok
*** testing nc_get_vars ... ok
*** testing nc_get_varm_text ... ok
*** testing nc_get_varm_uchar ... ok
*** testing nc_get_varm_schar ... ok
*** testing nc_get_varm_short ... ok
*** testing nc_get_varm_int ... ok
*** testing nc_get_varm_long ... ok
*** testing nc_get_varm_float ... ok
*** testing nc_get_varm_double ... ok
*** testing nc_get_varm_ushort ... ok
*** testing nc_get_varm_uint ... ok
*** testing nc_get_varm_longlong ... ok
*** testing nc_get_varm_ulonglong ... ok
*** testing nc_get_varm ... ok
*** testing nc_get_att_text ... 
	FAILURE at line 10854 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_get.c: value read not that expected
	### 1 FAILURES TESTING nc_get_att_text! ###
*** testing nc_get_att_uchar ... ok
*** testing nc_get_att_schar ... ok
*** testing nc_get_att_short ... ok
*** testing nc_get_att_int ... ok
*** testing nc_get_att_long ... ok
*** testing nc_get_att_float ... ok
*** testing nc_get_att_double ... ok
*** testing nc_get_att_ushort ... ok
*** testing nc_get_att_uint ... ok
*** testing nc_get_att_longlong ... ok
*** testing nc_get_att_ulonglong ... ok
*** testing nc_get_att ... 
	FAILURE at line 1689 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_read.c: buf read not that expected
	### 1 FAILURES TESTING nc_get_att! ###
*** testing nc_inq_att ... ok
*** testing nc_inq_attname ... ok
*** testing nc_inq_attid ... ok
*** testing nc_inq_attlen ... ok
*** testing nc_inq_atttype ... ok
*** testing nc_create ... ok
*** testing nc_redef ...  1386 good comparisons.  34 good comparisons.  1386 good comparisons.  34 good comparisons. ok
*** testing nc_sync ...  34 good comparisons.  1386 good comparisons. ok
*** testing nc_abort ...  1386 good comparisons.  34 good comparisons. ok
*** testing nc_def_dim ... ok
*** testing nc_rename_dim ... ok
*** testing nc_def_var ... ok
*** testing nc_put_var_text ... ok
*** testing nc_put_var_uchar ... ok
*** testing nc_put_var_schar ... ok
*** testing nc_put_var_short ... ok
*** testing nc_put_var_int ... ok
*** testing nc_put_var_long ... ok
*** testing nc_put_var_float ... ok
*** testing nc_put_var_double ... ok
*** testing nc_put_var_ushort ... ok
*** testing nc_put_var_uint ... ok
*** testing nc_put_var_longlong ... ok
*** testing nc_put_var_ulonglong ... ok
*** testing nc_put_var1_text ... ok
*** testing nc_put_var1_uchar ... ok
*** testing nc_put_var1_schar ... ok
*** testing nc_put_var1_short ... ok
*** testing nc_put_var1_int ... ok
*** testing nc_put_var1_long ... ok
*** testing nc_put_var1_float ... ok
*** testing nc_put_var1_double ... ok
*** testing nc_put_var1_ushort ... ok
*** testing nc_put_var1_uint ... ok
*** testing nc_put_var1_longlong ... ok
*** testing nc_put_var1_ulonglong ... ok
*** testing nc_put_var1 ...  1386 good comparisons. ok
*** testing nc_put_vara_text ... ok
*** testing nc_put_vara_uchar ... ok
*** testing nc_put_vara_schar ... ok
*** testing nc_put_vara_short ... ok
*** testing nc_put_vara_int ... ok
*** testing nc_put_vara_long ... ok
*** testing nc_put_vara_float ... ok
*** testing nc_put_vara_double ... ok
*** testing nc_put_vara_ushort ... ok
*** testing nc_put_vara_uint ... ok
*** testing nc_put_vara_longlong ... ok
*** testing nc_put_vara_ulonglong ... ok
*** testing nc_put_vara ...  1386 good comparisons. ok
*** testing nc_put_vars_text ... ok
*** testing nc_put_vars_uchar ... ok
*** testing nc_put_vars_schar ... ok
*** testing nc_put_vars_short ... ok
*** testing nc_put_vars_int ... ok
*** testing nc_put_vars_long ... ok
*** testing nc_put_vars_float ... ok
*** testing nc_put_vars_double ... ok
*** testing nc_put_vars_ushort ... ok
*** testing nc_put_vars_uint ... ok
*** testing nc_put_vars_longlong ... ok
*** testing nc_put_vars_ulonglong ... ok
*** testing nc_put_vars ...  1386 good comparisons. ok
*** testing nc_put_varm_text ... ok
*** testing nc_put_varm_uchar ... ok
*** testing nc_put_varm_schar ... ok
*** testing nc_put_varm_short ... ok
*** testing nc_put_varm_int ... ok
*** testing nc_put_varm_long ... ok
*** testing nc_put_varm_float ... ok
*** testing nc_put_varm_double ... ok
*** testing nc_put_varm_ushort ... ok
*** testing nc_put_varm_uint ... ok
*** testing nc_put_varm_longlong ... ok
*** testing nc_put_varm_ulonglong ... ok
*** testing nc_put_varm ...  1386 good comparisons. ok
*** testing nc_rename_var ...  1386 good comparisons. ok
*** testing nc_put_att_text ... ok
*** testing nc_put_att_uchar ... ok
*** testing nc_put_att_schar ... ok
*** testing nc_put_att_short ... ok
*** testing nc_put_att_int ... ok
*** testing nc_put_att_long ... ok
*** testing nc_put_att_float ... ok
*** testing nc_put_att_ulonglong ... ok
*** testing nc_put_att_ushort ... ok
*** testing nc_put_att_uint ... ok
*** testing nc_put_att_longlong ... ok
*** testing nc_put_att_ulonglong ... ok
*** testing nc_put_att ... 
	FAILURE at line 1085 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/util.c: nc_get_att_text: unexpected value
 33 good comparisons. 
	### 1 FAILURES TESTING nc_put_att! ###
*** testing nc_copy_att ...  34 good comparisons. ok
*** testing nc_rename_att ... 
	FAILURE at line 1859 of /build/netcdf/src/netcdf-c-4.6.0/nc_test/test_write.c: get_att_text: unexpected value
	### 1 FAILURES TESTING nc_rename_att! ###
*** testing nc_del_att ... ok
*** testing nc_against_pnetcdf ... ok
*** testing nc_set_default_format ... ok

*** Total number of failures: 12
*** nc_test FAILURE!!!
<end of output>
Test time =  10.93 sec
----------------------------------------------------------
Test Failed.
"nc_test" end time: Jan 25 22:14 CET
"nc_test" time elapsed: 00:00:10
----------------------------------------------------------

```

**Comment by ArchangeGabriel** [comment]
Should I just ignore this issue for now and package 4.6.1 anyway?

**Comment by WardF** [comment]
Hi, you should be able to ignore this for now.  I'll see what I can figure out about the failure :)

**Comment by ArchangeGabriel** [comment]
Seems fixed as of 4.7.1.
